src/main/java/com/direwolf20/buildinggadgets/client/gui/base/BasicGUIBase.java

```diff
@@ -0,0 +1,22 @@
+package com.direwolf20.buildinggadgets.client.gui.base;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiScreen;
+
+public class BasicGUIBase extends GuiScreen {
+
+    private static final char KEY_CLOSE_GUI = 'e';
+
+    @Override
+    public boolean charTyped(char character, int p_charTyped_2_) {
+        if (character == KEY_CLOSE_GUI)
+            Minecraft.getInstance().player.closeScreen();
```

Comment:
Which one do we prefer? `Minecraft.getInstance().displayScreen(null)` or `Minecraft.getInstance().player.closeScreen()`?

Reply:
`closeScreen`